orm/src/test/java/org/hibernate/search/test/interceptor/Blog.java

```diff
@@ -29,6 +31,18 @@ public void setId(Integer id) {
 		this.id = id;
 	}
 
+	@Override
+	public boolean equals(Object obj) {
+		return getClass().equals( obj.getClass() )
+				&& id != null
```

Comment:
Not worth a fix but we agree we don't need to test for the id to be not null here?